content/en/docs/concepts/scheduling-eviction/pod-scheduling-readiness.md

```diff
@@ -89,6 +89,24 @@ The metric `scheduler_pending_pods` comes with a new label `"gated"` to distingu
 has been tried scheduling but claimed as unschedulable, or explicitly marked as not ready for
 scheduling. You can use `scheduler_pending_pods{queue="gated"}` to check the metric result.
 
+## Mutable Pod Scheduling Directives
+
+{{< feature-state for_k8s_version="v1.27" state="beta" >}}
+
+Pods with scheduling gates have mutable scheduling directives, with certain constraints. 
+The rules for updating a Pod's scheduling directives are as follows:
```

Comment:
In the highlevel, you can only tighten the scheduling directives of a Pod. In other words, the updated directives would cause the Pods to only be able to schedule in a subset of the nodes that it would previously match. More explicitly, the rules for updating a Pod's scheduling directives are as follows: